regolith/tools.py

```diff
@@ -1287,4 +1287,15 @@ def get_id_from_name(coll, name):
                              case_sensitive=False)
     if person:
         return person["_id"]
-    else: return None
\ No newline at end of file
+    else: return None
+
+def search_collection(collection, arguments):
```

Comment:
please use TDD.  First the docstring, then the tests, then the code.

Reply:
Done.